docs/features.md

```diff
@@ -0,0 +1,21 @@
+# Features used and project decisions
+
+## List of features
+
+- `jQuery-3.3.1`: https://jquery.com/
+- `EJS`: https://ejs.co/
+- `ESLint`: https://eslint.org/
+- `Webpack`: https://webpack.js.org/
+- `@linx-impulse/commons-js`: https://github.com/chaordic/commons-js/
+- `Owl Carousel 2`: https://owlcarousel2.github.io/OwlCarousel2/
+- `Slick carousel`: http://kenwheeler.github.io/slick/
+- `Onsite-SDK`: https://github.com/chaordic/engage-onsite-sdk-js
+
+## Project decisions and some explanations
+
+- `@linx-impulse/commons-js`: reusing some util functions from this repo.
+- `Onsite-SDK`: SDK interface to use api.
+- `Owl Carousel 2`: plugin used to build the horizontal carousels.
+- `Slick carousel`: plugin used to build the vertical carousels.
+- The project has the components structures to the migration for a framework (such as React, Vue, Angular, etc) be more clear and easy.
```

Comment:
Trocaria a frase para:

`The project has the components structures for a clear and easy framework migration (such as React, Vue, Angular, etc).`

O que acha?